drivers/include/drivers/DigitalOut.h

```diff
@@ -18,8 +18,17 @@
 #define MBED_DIGITALOUT_H
 
 #include "platform/platform.h"
+
 #include "hal/gpio_api.h"
 
+/**
+ * Option to make certain class functions virtual for advanced polymorphism.
+ * Disabled by default for size and speed optimization.
+ */
+#ifndef MBED_CONF_DRIVERS_VIRTUALIZE_DIGITALOUT
+#define MBED_CONF_DRIVERS_VIRTUALIZE_DIGITALOUT false
```

Comment:
This isn't needed - `#if XXX` is treated as `#if 0` if it's not defined.

(And I'm not sure why it wouldn't be defined).